back-end/requests/put/add_members.js

```diff
@@ -0,0 +1,43 @@
+let Playlist = require("../../models/playlists.model");
+
+const add_member = async (req, res) => {
```

Comment:
this should take the arguments `(req, res, next)`, as `next` will be needed for error handling